pvlib/clearsky.py

```diff
@@ -65,22 +65,22 @@ def ineichen(apparent_zenith, airmass_absolute, linke_turbidity,
     References
     ----------
     [1] P. Ineichen and R. Perez, "A New airmass independent formulation for
-        the Linke turbidity coefficient", Solar Energy, vol 73, pp. 151-157,
-        2002.
+    the Linke turbidity coefficient", Solar Energy, vol 73, pp. 151-157,
+    2002.
```

Comment:
Neither the current format nor the proposed change is consistent with references section of [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard)

Reply:
Fair, but I am having zero success with doing it the "right way" and getting the results to look any better than the ones here: https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.atmosphere.first_solar_spectral_correction.html

Is that how references are supposed to look?

Reply:
Hm... They *should* look like the format in the numpydoc example. I don't know why they do not. I poked at the html/css. I think numpydoc is writing out the correct rst and sphinx is processing it into the correct html, so I'm guessing it's related to the css. It could be specific to the readthedocs sphinx theme, but that's just a guess. 

So, none of the formats are ideal. I have a slight preference for sticking with the numpydoc format and hoping that someone, somewhere, fixes some bug for us. But I'm also ok with your "plain text" approach.

Reply:
> It could be specific to the readthedocs sphinx theme, but that's just a guess.

Good guess!  Looks like it's an incompatibility between sphinx_rtd_theme and sphinx 2.0+: https://github.com/readthedocs/sphinx_rtd_theme/issues/741.  It's a bit of a rabbit hole, but I've found two workarounds:

- Using sphinx <2.0 fixes the reference rendering, or at least v1.8.5 does for me locally.  Interestingly, RTD is installing 1.8.5 first and then replacing it with 2.2.2 -- see [the most recent RTD build log](https://readthedocs.org/api/v2/build/10102768.txt), ctrl-F "Successfully uninstalled Sphinx-1.8.5".  I'm guessing that the `sphinx=1.7.2` in the `environment.yml` file isn't being respected because that file is used for conda installs, but `readthedocs.yml` says to use pip and install all extra requirements, which according to `setup.py` includes sphinx without a version constraint.  So I'd guess that pinning sphinx at a pre-2.0 version in `setup.py` would change the version RTD ends up using.  RTD also encourages pinning specific versions of sphinx, see the blue `Note` section here: https://docs.readthedocs.io/en/stable/builds.html#understanding-what-s-going-on
- Apparently sphinx 2.0+ switched to HTML5 output, but the old HTML4 writer is [still available](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html4_writer).  I tried it out locally and it also fixes the references problem.

Both of these are a little heavy handed and have an effect on how the rest of the docs are rendered so I wanted to get feedback on whether this seemed like a good idea or if you'd prefer to just wait for the theme to fix it on their end.  Either way, I'll standardize the docstring references to the numpydoc format.  Some other functions already had the plaintext version (e.g. [clearsky.bird](https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.clearsky.bird.html)) so I'll change those as well.

Reply:
> standardize the docstring references to the numpydoc format.
+1 here.

Reply:
You are correct about the conda file being ignored. I’m ok with either work around.